### PR TITLE
update README

### PR DESCRIPTION
**Pull request by rhagarty**
still have some issues:
- section #5 HDFS data. Why do wget locally?
- ambari services screen shot - right services?
- livy screen shot show too many services

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,39 +47,53 @@ When you have completed this Code Pattern, you will understand how to:
...
+> 1. This code pattern requires that version `2.2.0` of the `Spark2` service be installed.
+> 2. Details on how to install and configure the `Solr` service are discussed in the [Setup HDP Search](#1-set-up-hdp-search) section below.
+
+## Disable Apache Livy CSRF protection
+
+From your Ambari Server UI, disable the Apache Livy server's CSRF proctection by changing the `livy.server.csrf_protection.enable` variable to false, as shown here:
+
+![](images/dsx-local-create-project-spark2-livy.png)
+
+To navigate to this panel, click on the `Spark2` service, then the `Configs` tab, and then open the `Advanced livy2-conf` section.
```
Comment (stevemar) [inline README.md:76]: nit pick for readability here, make the thing that the user does first, appear first. logically, this should go earlier...

```
We can disable the Apache Livy server's CSRF protection from the Ambari Server UI. To do this, in the console click on the `Spark2` service, then the `Configs` tab, and then open the `Advanced livy2-conf` section. Now we can choose to disable the Livy server's CSRF protection by changing `livy.server.csrf_protection.enable` to `false`. See the example below.
```

**Review by stevemar — APPROVED** [review]
this looks fantastic.

**Comment by stevemar** [comment]
@dilipbiswal please review ASAP.

**Comment by dilipbiswal** [comment]
LGTM

**Pull request merged**